src/Fixtures/Crawlers.php

```diff
@@ -1342,6 +1350,7 @@ class Crawlers extends AbstractProvider
         'ZnajdzFoto',
         'ZnHTTP',
         'Zombie\.js',
+        'ZoominfoBot',
```

Comment:
We already detect this as it has the string `bot` in it.

If you remove it, the tests should then pass.

Thanks 👍

Reply:
I removed this Crawaler but it doesn't show up as crawler before. Curious.

Reply:
Strange, we definitely detect it as a bot, see here - https://crawlerdetect.io/?q=ZoominfoBot